build/filters/tag.js

```diff
@@ -107,5 +107,27 @@ module.exports = [
 				}
 			}
 		});
+	},
+
+	/**
+	 * Set image to tag. If tag name is an img, it will be set as src
```

Comment:
Sets an image to a tag. If the tag name is img, it will be set as src
otherwise it will be set as background-image in a style attribute.